pubspec.yaml

```diff
@@ -1,20 +1,38 @@
 name: crypto
-version: 2.2.0-nullsafety-dev
+version: 3.0.0-dev
 description: Implementations of SHA, MD5, and HMAC cryptographic functions
 homepage: https://www.github.com/dart-lang/crypto
 
 environment:
-  sdk: '>=2.10.0-2.0.dev <2.11.0'
+  sdk: '>=2.12.0-dev <3.0.0'
 
 dependencies:
   collection: ^1.15.0-nullsafety
-  convert: ^2.2.0-nullsafety
+  convert: ^3.0.0-nullsafety
   typed_data: ^1.3.0-nullsafety
 
 dev_dependencies:
   pedantic: ^1.10.0-nullsafety
   test: ^1.16.0-nullsafety
 
 dependency_overrides:
+  analyzer:
```

Comment:
you should be able to get rid of all of these...right?

Or at least use pub overrides and not git overrides

Reply:
+1 on using pub overrides